cluster/router/tag/tag_router.go

```diff
@@ -55,6 +64,15 @@ func (c *tagRouter) isEnabled() bool {
 	return c.enabled
 }
 
+func (c *tagRouter) SetApplication(app string) {
+	c.application = app
+}
+
+func (c *tagRouter) tagRouterRuleCopy() RouterRule {
+	routerRule := *c.tagRouterRule
+	return routerRule
```

Comment:
```suggestion
	return *c.tagRouterRule
```

enough? and i think no need to extract a new function for single scene